code/modules/mob/living/emote.dm

```diff
@@ -720,9 +720,14 @@
 	message = "beeps."
 	message_param = "beeps at %t."
 	sound = 'sound/machines/twobeep.ogg'
-	mob_type_allowed_typecache = list(/mob/living/brain, /mob/living/silicon, /mob/living/basic/orbie)
+	mob_type_allowed_typecache = list(/mob/living/carbon/human, /mob/living/brain, /mob/living/silicon, /mob/living/basic/orbie)
```

Comment:
You know if we wanted to we could just nuke the typecache entirely and give the trait to silicons/brains/orbs in `Initialize` as an `INNATE_TRAIT`.

Food for thought

Reply:
Not a bad thought... I'll think about it

Reply:
thought about it-